README.md

```diff
@@ -148,6 +148,9 @@ docker run --rm -it -v "$(pwd)":/paste jkcclemens/paste
 By default, that will build paste, but you can also append `/bin/bash` to the command to start a
 Bash shell in the working environment.
 
+## API
+See the [paste API documentation](api.md)
```

Comment:
Can I get a period at the end? <3

Reply:
I don't know about that. Should an exclamation mark be considered?

Reply:
Perhaps confuse the users instead with an interrobang‽

Reply:
I think we should reword it to be less demanding, maybe "If you would like to see the API docs, please have a look at [...]". Maybe they don't really care about it.

Reply:
"If you wish to read the API documentation, you are invited to view it [here](api.md) but should not feel any pressure to do so. You are wonderful just the way you are."

Reply:
It's really missing a heart, @mbax

Reply:
OMG HE RESPONDED TO ME :heart:

Reply:
>Can I get a period at the end? <3

They're called full-stops